### DISCUSSION TITLE
How do build i my tailwind config with every class enabled? (to speed up development)

### DISCUSSION THREAD

**Discussion by gabrielesilinic** [tailwindlabs/tailwindcss#7737] [Help]
i will never use this in production, but i really need to find a way to do this

i know that there is cdn.tailwindcss.com but i need to build a specific configuration that doesn't work via cdn anymore, because i'm using tailwind thogether with daisyui (wonderful plugin btw), if i use the cdn tailwind's prefixes do not work with classes from daisyui
\
\
\
\
btw, tailwind together with daisyui are a blessing, thanks for making this framework public and opensource  (since i'm not a wonderful but only a decent UI designer those frameworks to build wonderful UIs anyway, i'm a programmer, UI design it's just not my main ability even if i like doing also that)

edit-notes:
i'm using the recommended configuration, npm with tailwind and postcss and various tailwind plugins

**Comment by adamwathan** [comment]
Check this discussion for some information:

https://github.com/tailwindlabs/tailwindcss/discussions/6256

No built in way to generate a build that contains *every* class — the combination of possible classes in Tailwind 3 is just too high. We tried an experiment to see what would happen if we tried and node crashed after the CSS file got to 4gb in size.

**Reply by gabrielesilinic** [reply]
Any way to pass my configuration to the cdn? (Or to self host something similar locally in loopback just for testing so i don't stress tailwind's servers)

**Reply by mflorida** [reply]
You could download the hosted code to a file and include that in your build or add via `<script>` tag.

https://cdn.tailwindcss.com/3.0.23
